dask/dataframe/core.py

```diff
@@ -5138,17 +5138,20 @@ def cov_corr_chunk(df, corr=False):
         with warnings.catch_warnings(record=True):
             warnings.simplefilter("always")
             mu = (sums / counts).T
-        m = np.zeros(shape)
+        m = np.zeros_like(df.values, shape=shape)
```

Comment:
I think that, for compatibility with older numpy versions, you probably need to use this:

https://github.com/dask/dask/blob/539d1e27a8ccce01de5f3d49f1748057c27552f2/dask/array/utils.py#L325-L334

Reply:
Ah, good call - Just noticed that the `shape` argument is so new